libs/nx-plugin/src/executors/code-gen/schema.json

```diff
@@ -31,7 +31,7 @@
             "x-prompt": "What is the name of the project you want to add the component to?"
         },
 
-        "featureBoardProjectName": {
+        "featureBoardProductName": {
             "type": "string",
             "description": "The name of the FeatureBoard project you want to pull the types from."
```

Comment:
Change to:
The name of the FeatureBoard product you want to pull the types from.

Reply:
Does "description" above also need to change? "The name of the project to add the component to."